docs/api/actions/type.mdx

```diff
@@ -618,22 +634,23 @@ following:
-| [5.6.0](/guides/references/changelog#5-6-0)   | Support single key combination syntax                                                                                     |
-| [5.5.0](/guides/references/changelog#5-5-0)   | Support `beforeinput` event                                                                                               |
-| [3.4.1](/guides/references/changelog#3-4-1)   | Added `parseSpecialCharSequences` option                                                                                  |
-| [3.3.0](/guides/references/changelog#3-3-0)   | Added `{insert}`, `{pageup}` and `{pagedown}` character sequences                                                         |
-| [3.2.0](/guides/references/changelog#3-2-0)   | Added `{home}` and `{end}` character sequences                                                                            |
-| [0.20.0](/guides/references/changelog#0-20-0) | Supports for typing in inputs of type `date`, `time`, `month`, and `week`                                                 |
-| [0.17.1](/guides/references/changelog#0-17-1) | Added `ctrl`, `cmd`, `shift`, and `alt` keyboard modifiers                                                                |
-| [0.16.3](/guides/references/changelog#0-16-3) | Supports for typing in elements with `tabindex` attribute                                                                 |
-| [0.16.2](/guides/references/changelog#0-16-2) | Added `{downarrow}` and `{uparrow}` character sequences                                                                   |
-| [0.8.0](/guides/references/changelog#0-8-0)   | Outputs Key Events Table to console on click                                                                              |
-| [0.8.0](/guides/references/changelog#0-8-0)   | Added `{selectall}`, `{del}`, `{backspace}`, `{esc}`, `{{}`, `{enter}`, `{leftarrow}`, `{rightarrow}` character sequences |
-| [0.8.0](/guides/references/changelog#0-8-0)   | Added small delay (10ms) between each keystroke during `cy.type()`                                                        |
-| [0.6.12](/guides/references/changelog#0-6-12) | Added option `force`                                                                                                      |
+| Version                                         | Changes                                                                                                                                |
+| ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| [13.14.0](/guides/references/changelog#13-14-0) | Added support for `.type({upArrow})` and `.type({downArrow})` to work on date, month, week, time, datetime-local and range input types |     |
```

Comment:
Looks like there is an extra column here?

```suggestion
| ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
| [13.14.0](/guides/references/changelog#13-14-0) | Added support for `.type({upArrow})` and `.type({downArrow})` to work on date, month, week, time, datetime-local and range input types |
```

Reply:
good catch. This is broken 
<img width="1296" alt="Screenshot 2024-08-27 at 1 27 30 PM" src="https://github.com/user-attachments/assets/50ff4f8b-8489-4a9c-a247-b927e858fba8">

Reply:
fixed 
<img width="1296" alt="Screenshot 2024-08-27 at 1 29 34 PM" src="https://github.com/user-attachments/assets/15e4e6de-93ab-4ed9-8b81-80bc5f38eb97">